apps/webapp/app/v3/services/resumeAttempt.server.ts

```diff
@@ -78,6 +87,8 @@ export class ResumeAttemptService extends BaseService {
         return;
```

Comment:
What happens to the run attempt when this function returns undefined?

Reply:
We don't currently do anything with the return value as resumes are two separate messages. Tasks saying ready for resume, platform sending the resume data. There's room to fail runs in a bunch of places (and shut down the workers), but it's tough because a single no-op message could mean the entire run fails.

Reply:
Makes sense, let's leave it for now then I think until we are more certain on the real world solution to this (if there even is a problem)